test/IISExpress.FunctionalTests/WindowsAuthTests.cs

```diff
@@ -26,7 +26,8 @@ public static TestMatrix TestVariants
             => TestMatrix.ForServers(DeployerSelector.ServerType)
                 .WithTfms(Tfm.NetCoreApp22, Tfm.Net461)
                 .WithAllApplicationTypes()
-                .WithAllAncmVersions();
+                .WithAllAncmVersions()
+                .WithAllHostingModels();
 
         [ConditionalTheory]
         [MemberData(nameof(TestVariants))]
```

Comment:
The actual test should confirm which response is expected based on which hosting model. It looks like in-proc would return "backcompat;..."